ProjectShop/UserApp/templates/UserApp/profile.html

```diff
@@ -16,5 +16,6 @@ <h2>{% translate "Profile Info" %}</h2>
         {{ form|crispy }}
 
         <button class="btn btn-outline-info" type="submit">{% translate "Update" %}</button>
+        <a href="{% url 'password_change' %}">{% translate "Change Password?" %}</a>
```

Comment:
people often use "Forgot Password?" instead. maybe it`s your own style?

Reply:
Thank you for your feedback. I appreciate it. The idea is that a user may change his\her password. Perhaps, the question mark is abundant. Tomorrow, I'd like to discuss that with the team.

![Screenshot from 2021-10-05 22-19-08](https://user-images.githubusercontent.com/56480204/136090115-4e280612-7ed9-4fd4-a546-850b80ff39e3.png)